imgaug specifies a requirement for opencv-python-headless which causes conflicts for other installations of OpenCV
this patch removes the requirement, and will work as long as we load an appropriate (standard) OpenCV module
See https://github.com/aleju/imgaug/issues/473

author: Andrew Edmondson (University of Birmingham)

--- setup.py.orig	2024-03-04 09:20:57.356025008 +0100
+++ setup.py	2024-03-04 09:22:09.182207006 +0100
@@ -15,16 +15,11 @@
     "Pillow",
     "matplotlib",
     "scikit-image>=0.18",
-    "opencv-python-headless",
     "imageio<=2.6.1; python_version<'3.5'",
     "imageio; python_version>='3.5'",
     "Shapely"
 ]
 
-ALT_INSTALL_REQUIRES = {
-    "opencv-python-headless": ["opencv-python", "opencv-contrib-python", "opencv-contrib-python-headless"],
-}
-
 
 def check_alternative_installation(install_require, alternative_install_requires):
     """If some version version of alternative requirement installed, return alternative,
@@ -54,8 +49,6 @@
     return install_requires
 
 
-INSTALL_REQUIRES = get_install_requirements(INSTALL_REQUIRES, ALT_INSTALL_REQUIRES)
-
 setup(
     name="imgaug3",
     version="0.4.1",
